apps/playground-api/src/setups/setup.ts

```diff
@@ -33,10 +33,6 @@ export abstract class PlaygroundSetup<Each> {
     await this.before(list)
 
     for (const each of list) {
-      if (await this.has(each)) {
```

Comment:
@canonbrother Here, I changed your code.  As this has function always returns false, this code can be removed.

Reply:
don't change that code.. its by designed

Reply:
Okay